hack/release.toml

```diff
@@ -120,6 +120,7 @@ vlan=eth1.5:eth1 ip=172.20.0.2::172.20.0.1:255.255.255.0::eth1.5:::::
 * Linux: 5.15.59
 * Flannel 0.19.1
 * containerd 1.16.7
+* Kubernetes: v1.25.0-beta.0
```

Comment:
I think we need to add a note on no longer labeling control plane nodes with `master` label, and changing the taint to be `control-plane` now. This is a breaking change for some workloads